static/js/types.ts

```diff
@@ -0,0 +1,10 @@
+// TODO: Move this to server_events_dispatch
```

Comment:
Maybe we should start writing these typescript TODOs as `TODO/typescript: Move ...` so that it's easy to grep for them?  We've used that convention in a few places.

Reply:
Done.